src/main/java/org/java_websocket/drafts/Draft_6455.java

```diff
@@ -525,7 +525,8 @@ private Framedata translateSingleFrame( ByteBuffer buffer ) throws IncompleteExc
      * @throws LimitExceededException if the payload length is to big
      */
     private TranslatedPayloadMetaData translateSingleFramePayloadLength(ByteBuffer buffer, Opcode optcode, int oldPayloadlength, int maxpacketsize, int oldRealpacketsize) throws InvalidFrameException, IncompleteException, LimitExceededException {
-        int payloadlength = oldPayloadlength, realpacketsize = oldRealpacketsize;
+        int payloadlength = oldPayloadlength,
```

Comment:
Are you sure that fixed the code style issue? 🤔 Probably meant to put them in separate declarations

Reply:
Does the magic ;) (only when we are in a non strict mode according to the documentation)